.gitignore

```diff
@@ -3,3 +3,6 @@ build
 composer.lock
 files
 vendor
+/.buildpath
```

Comment:
These changes belong to your IDE and shouldn't be in project but your local global gitignore.